Integrations/FireEyeHelix/FireEyeHelix.py

```diff
@@ -0,0 +1,1643 @@
+        params = assign_params(
+            limit=limit,
+            offset=offset,
+            sort=sort
+        )
+        return self._http_request('GET', suffix, params=params)
+
+    def edit_rule(self, rule_id: Optional[Any], enabled: Union[str, bool] = None) -> Dict:
+        """Edit a single rule using PATCH request
+
+        Args:
+            rule_id: ID of the rule.
+            enabled: Is the rule enabled.
+
+        Returns:
+            Response from API
+        """
+        suffix = f'/api/v1/rules/{rule_id}'
+        body = assign_params(enabled=enabled != 'false' if enabled else None)
+        return self._http_request('PATCH', suffix, json_data=body)
+
+    def add_list_item(self, list_id: Union[str, int], type: str, value: str, risk: str = None,
+                      notes: str = None) -> Dict:
+        """Adds a single item list to a list
+
+        Args:
+            list_id: List id.
+            type: Type of list item.
+            value: Value of list item.
+            risk: Risk of list item.
+            notes: Notes for list item.
+
+        Returns:
+            Respone from API
+        """
+        suffix = f'/api/v3/lists/{list_id}/items'
+        body = assign_params(
+            type=type,
+            value=value,
+            risk=risk,
+            notes=notes
+        )
+        return self._http_request('POST', suffix, json_data=body)
+
+    def update_list_item(self, list_id: Union[str, int], item_id: Union[str, int], type: str = None, value: str = None,
+                         risk: str = None, notes: str = None) -> Dict:
+        """Updates a single item list
+
+        Args:
+            list_id: List id.
+            item_id: Item id.
+            type: Type of list item.
+            value: Value of list item.
+            risk: Risk of list item.
+            notes: Notes for list item.
+
+        Returns:
+            Respone from API
+        """
+        suffix = f'/api/v3/lists/{list_id}/items/{item_id}'
+        body = assign_params(
+            type=type,
+            value=value,
+            risk=risk,
+            notes=notes
+        )
+        return self._http_request('PATCH', suffix, json_data=body)
+
+    def get_list_items(self, list_id: Optional[Any], offset: Optional[Any]) -> Dict:
+        """Gets items of a list
+
+        Args:
+            list_id: List ID.
+            offset: Offset in database.
+
+        Returns:
+            Response from API
+        """
+        suffix = f'/api/v3/lists/{list_id}/items'
+        params = assign_params(offset=offset)
+        return self._http_request('GET', suffix, params=params)
+
+    def remove_list_item(self, list_id: Optional[Any], item_id: Optional[Any]) -> Dict:
+        suffix = f'/api/v3/lists/{list_id}/items/{item_id}'
+        return self._http_request('DELETE', suffix, resp_type='content')
+
+
+''' HELPER FUNCTIONS '''
+
+
+def build_transformed_dict(src: Union[Dict, List], trans_dict: Dict) -> Union[Dict, List]:
```

Comment:
is it really necessary to get and return Union here?
This my be a sign that this function is too general and may need to be splitter to 1 or more different functions.

Reply:
Well it works like `tableToMarkdown` where it's a general implementation for both a single result and multiple results. changing it will require adding a for loop for each commands that calls it in case the result that was returned is a list. I can do this, but this will require further testing as this version of the command works in other integration I've written